### PR TITLE
Add OTLP Metric ADOT Operator Test

### PR DESCRIPTION
**Pull request by sethAmazon**

### CODE REVIEW

**Comment by sethAmazon** [comment]
Trace and Prometheus test cases have already been added. Thus only adding metric

File: terraform/testcases/otlp_metric_adot_operator/otconfig.tpl [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      awsemf:
+        region: '${region}'
+
+    service:
+      pipelines:
+        metrics:
+          receivers: [otlp]
+          processors: [batch]
+          exporters: [logging,awsemf]
+      extensions: [pprof]
```
Comment (mxiamxia) [inline terraform/testcases/otlp_metric_adot_operator/otconfig.tpl:25]: it looks the yaml content is not well formatted as well in this PR

Comment (sethAmazon) [reply terraform/testcases/otlp_metric_adot_operator/otconfig.tpl:25]: I changed the formatting to match the otlp_trace_adot_operator. It works now.

Comment (sethAmazon) [reply terraform/testcases/otlp_metric_adot_operator/otconfig.tpl:25]: I think the formatter is putting this file at the begging of line thus we need a double indent. https://github.com/aws-observability/aws-otel-test-framework/blob/terraform/terraform/eks/adot-operator/adot_collector_deployment.tpl https://github.com/aws-observability/aws-otel-test-framework/blob/terraform/terraform/testcases/otlp_trace_adot_operator/otconfig.tpl

**Review by mxiamxia — APPROVED** [review]

**Pull request merged**